src/plugins/cosmos/index.tsx

```diff
@@ -44,14 +45,38 @@ export function register(): Plugins {
             hide: true,
             label: '',
             main: () => <CosmosAsset chainId={'cosmos:osmosis-1'} />,
-            icon: <AssetIcon src='https://assets.coincap.io/assets/icons/atom@2x.png' />
+            icon: <AssetIcon src='https://assets.coincap.io/assets/icons/atom@2x.png' />,
+            routes: [
+              {
+                path: '/',
+                label: 'navBar.overview',
+                main: () => <CosmosAsset chainId={'cosmos:osmosis-1'} />
+              },
+              {
+                path: '/transactions',
+                label: 'navBar.transactions',
+                main: () => <CosmosAssetTxHistory chainId={'cosmos:cosmoshub-4'} />
```

Comment:
```suggestion
                main: () => <CosmosAssetTxHistory chainId={'osmos:osmosis-1'} />
```

Reply:
My bad, looks like I forgot to test Cosmos on Osmosis transactions tab, it's fixed now 👍